test-bench.sh

```diff
@@ -583,17 +592,17 @@ function merge_task(){
 
     info "Operation: merge"
     if [ -z "${2}" ]; then
-      file_list=$( ls "${TARGET}/jmeter/results/"*AGGREGATE.csv 2> /dev/null )
+      file_list=$( ls "${SDKMS_REST_API_HOME}/target/jmeter/results/"*AGGREGATE.csv 2> /dev/null )
     else
       IFS=',' read -r -a file_list <<< "$2"
     fi
     # check file_list is empty
-    if [ -z "${file_list}" ]; then
-        error "No aggergated csv file found"
+    if [ -z "$file_list" ]; then
```

Comment:
it is better to use ${VAR} in bash

Reply:
Sure. Will club this feature in a separate PR since I would like to test it.